recipes/fftw/build.sh

```diff
@@ -0,0 +1,9 @@
+# Install single precision
+./configure --prefix=$PREFIX --exec-prefix=$PREFIX --enable-shared --disable-static --enable-single
+make
+make install
+# Install double precision
+./configure --prefix=$PREFIX --exec-prefix=$PREFIX --enable-shared --disable-static
+make
+make install
```

Comment:
So, we actually have FFTW. Though if you are interested in FFTW, we would love to have you as a maintainer. Here is the [feedstock]( https://github.com/conda-forge/fftw-feedstock ). If you wish to join, just add your GitHub handle to the [maintainers' list]( https://github.com/conda-forge/fftw-feedstock/blob/master/recipe/meta.yaml#L31-L32 ) in a PR.